src/platform/forms/components/FormFooter.jsx

```diff
@@ -1,19 +1,31 @@
 import React from 'react';
 
-import AskVAQuestions from './AskVAQuestions';
-
 export default class FormFooter extends React.Component {
   render() {
     const { formConfig, currentLocation } = this.props;
     const GetFormHelp = formConfig.getHelp;
```

Comment:
Should we either pass the actual formConfig in here (which would have the getHelp property set on it) or only pass the getHelp component in directly as a getHelp prop (rather than nest it in a formConfig prop)?

Reply:
I think the `footerContent` config property is passed `formConfig` in USFS, so I don't think we have a choice here.